velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -68,20 +68,20 @@ void WindowFuzzer::addWindowFunctionSignatures(
   }
 }
 
-std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
-  auto frameType = [](int value) -> const std::string {
+std::tuple<std::string, bool> WindowFuzzer::generateFrameClause(
+    const core::WindowNode::WindowType& typeOfFrame,
```

Comment:
Nit : Rename to frameType. We usually try to avoid prepositions like Of in variable names.